department-of-veterans-affairs/va.gov-cms
# VACMS-4355 VC facility health service

## Description
See #4355 

## Testing done
Visual, behat, phpunit

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/108557311-634cca00-72c6-11eb-9b69-ebdfb75b23a9.png)

## QA steps
- As an administrator, go to `node/add/vet_center_facility_health_servi`
- Select Boston Vet Center, and choose the Complementary and alternative medicine options.
- Save the form, and visually verify the page and path match the screenshot, above.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?

- [x] `DO NOT MERGE` prior to https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4366 being merged

### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
